### PR TITLE
Fix AWS enum in schema

### LINKED ISSUE #248: Schema for Processors does not recognise AWS
**Describe the bug**
Typo in schema enum for AWS (event router)

### PR DESCRIPTION
**Pull request by embano1**
## Summary

Fix `enum` for AWS not matching const impacting schema validation.

Signed-off-by: Michael Gasch <mgasch@vmware.com>

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [x] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commmits related to your change
- [x] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [ ] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

* Closes #248

## Testing Verification

Unit/integration tests pass.

## Additional Information

n/a

### CODE REVIEW

File: vmware-event-router/routerconfig.schema.json [hunk 1]
```diff
@@ -1 +1 @@
...
...e":"boolean","description":"Enable checkpointing via checkpoint file for event recovery and replay purposes"},"checkpointDir":{"type":"string","description":"Directory where to persist checkpoints if enabled","default":"./checkpoints"},"auth":{"$schema":"http://json-schema.org/draft-04/schema#","$ref":"#/definitions/AuthMethod","description":"Authentication configuration for this section"}},"additionalProperties":false,"type":"object","oneOf":[{"required":["auth"],"title":"auth"}]},"RouterConfig":{"required":["apiVersion","kind","metadata","eventProvider","eventProcessor","metricsProvider"],"properties":{"apiVersion":{"enum":["event-router.vmware.com/v1alpha1"],"type":"string"},"kind":{"enum":["RouterConfig"],"type":"string"},"metadata":{"$schema":"http://json-schema.org/draft-04/schema#","$ref":"#/definitions/ObjectMeta"},"eventProvider":{"$schema":"http://json-schema.org/draft-04/schema#","$ref":"#/definitions/Provider"},"eventProcessor":{"$schema":"http://json-schema.org/draft-04/schema#","$ref":"#/definitions/Processor"},"metricsProvider":{"$schema":"http://json-schema.org/draft-04/schema#","$ref":"#/definitions/MetricsProvider"}},"additionalProperties":false,"type":"object"}}}
```
Comment (codegold79) [inline vmware-event-router/routerconfig.schema.json:1]: When I do a search for `awsEventBridge`, I still see three instances that are still in camel case form. Should these be changed to snake case for consistency?

**Review by codegold79 — COMMENTED** [review]
Had a question.

File: vmware-event-router/routerconfig.schema.json (hunk 1 above)
Comment (embano1) [reply vmware-event-router/routerconfig.schema.json:1]: I think it has to remain camelCase due to the reference in the JSON/YAML (struct tags)

```go
yaml:"awsEventBridge,omitempty" json:"awsEventBridge,omitempty" jsonschema:"oneof_required=awsEventBridge"
```

`awsEventBridge` is the field name and thus reference in the schema so I think the oneOf has to remain camelCase.

**Review by codegold79 — APPROVED** [review]
LGTM

**Pull request merged**